### DISCUSSION TITLE
How to enable GraphQL  playground?

### DISCUSSION THREAD

**Discussion by rpateld** [prisma/prisma#2613] [Q&A] [answered]
How to enable GraphQL  playground?

**Comment by hi019** [comment]
You mean for modifying data directly? I don't think Prisma clients offer this functionality, but the Prisma Engine does accept queries via GraphQL https://github.com/prisma/prisma-engines

**Reply by Sytten** [reply]
This is an implementation detail now though, likely to be changed.

**Comment by ryands17** [comment, marked as answer]
Hey @rpateld :wave: 
You can start run this [example](https://github.com/prisma/prisma-examples/tree/master/javascript/graphql) locally and you can open the GraphQL Playground on [http://localhost:4000](http://localhost:4000) which is the default in these examples.

**Comment by rpateld** [comment]
Thanks for reply.  For this example [graphql-nextjs](https://github.com/prisma/prisma-examples/tree/master/typescript/graphql-nextjs) GraphQL Playground not coming.

**Reply by ryands17** [reply]
@rpateld Currently the Playground isn't enabled in the example, we will add that soon. Till then, you can use the [Chrome extension](https://chrome.google.com/webstore/detail/graphql-playground-for-ch/kjhjcgclphafojaeeickcokfbhlegecd?hl=en) for GraphQL Playground and enter the URL `http://localhost:3000/api`. It should show the schema in the Playground.

**Reply by ekley** [reply]
@rpateld before this when Prisma was using Docker, I could've run Prisma playground locally on top of docker and directly query data base based on Prisma api, but now how can  I run Prisma Playground?

Note: when I run prisma migrate dev after it's done I see below log in the console :
'PostgreSQL database "MyDBName", schema "public" at "localhost:5432"'

Previously (in older version of Prisma) I could use Prisma playground on this -> http://localhost:5432 url

**Reply by janpio** [reply]
This does not exist in Prisma 2+ any more, you have to build your own GraphQL server on top of the Prisma ORM that can then offer a GraphQL Playground for users.

**Comment by ludralph** [comment]
Hi @rpateld,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
